### PR TITLE
Port docs- commands to Annotated

### PR DESCRIPTION
**Pull request by weitzman**

### CODE REVIEW

File: commands/core/docs.drush.inc [hunk 1]
```diff
@@ -1,310 +0,0 @@
...
-  );
-  $items['docs-aliases'] = array(
-    'description' => 'Site aliases overview on creating your own aliases for commonly used Drupal sites with examples from example.aliases.drushrc.php.',
-    'hidden' => TRUE,
-    'topic' => TRUE,
-    'bootstrap' => DRUSH_BOOTSTRAP_NONE,
-    'callback' => 'drush_print_file',
-    'callback arguments' => array($docs_dir . '/examples/example.aliases.drushrc.php'),
-  );
-  $items['docs-ini-files'] = array(
```
Comment (jibran) [inline commands/core/docs.drush.inc]: I saw that we removed the example file but `\Drush\Commands\core\EditCommands::phpIniFiles` still supports `DRUSH_INI`, `/etc/drush.ini` and `~/.drush/drush.ini`.

Should we add `example.drush.ini` back?

Comment (weitzman) [reply commands/core/docs.drush.inc]: no, those files are no longer used. we need to fix EditCommands.

Comment (weitzman) [reply commands/core/docs.drush.inc]: See 2nd tip at http://docs.drush.org/en/master/install/

Comment (jibran) [reply commands/core/docs.drush.inc]: So how are we going to update drush specific php config? Can we add php ini override in drush.yml?

Comment (greg-1-anderson) [reply commands/core/docs.drush.inc]: What is your use-case for drush-specific php config?

Comment (jibran) [reply commands/core/docs.drush.inc]: To setup the memory limit `memory_limit = 256M`

**Pull request merged**